### ISSUE TITLE
[All Versions][Compatibility ]: (Let's Do) Beachparty sand slabs

### ISSUE THREAD

**Issue by Numerosityy** [JackGlobetrotter/Full-Slabs#17]
### Full-Slabs Version

All Versions

### Minecraft Version

All Versions

### Fabric Loader Version

All Versions

### Fabric API Version

All Versions

### What happened?

Regardless of version, Let's Do Beachparty's Sand slabs aren't compatible with the mod.

### latest.log

No notable lines relating to either mod

### Did you isolate Full-Slabs?

I tried Full-Slabs alone

**Comment by JackGlobetrotter** [comment]
Hey, please provide latest.log! Did you try running only your mod and full-slab? What exactly do you mean by incompatible?

**Comment by Numerosityy** [comment]
When I say incompatible, I mean the sand slabs cannot be mixed nor placed vertically. The log doesn't seem to provide any helpful info.

It might be that the block is missing a slab tag that this mod uses?

**Comment by JackGlobetrotter** [comment]
Indeed, the mod does not define the "slab" as a slab block. As such, the block misses the required tags to work with this mod, sorry !
